api/src/main/java/org/openmrs/module/ipd/api/service/impl/WardServiceImpl.java

```diff
@@ -35,7 +36,15 @@ public WardPatientsSummary getIPDWardPatientSummary(String wardUuid) {
     @Override
     public List<AdmittedPatient> getWardPatientsByUuid(String wardUuid) {
         Location location= Context.getService(LocationService.class).getLocationByUuid(wardUuid);
-        return wardDAO.getAdmittedPatientsByLocation(location);
+        return wardDAO.getAdmittedPatients(location,null, null);
+    }
+
+    @Override
+    public List<AdmittedPatient> getProviderWardPatientsByUuid(String wardUuid, String providerUuid) {
```

Comment:
can we rename it to getPatientsByWardAndProvider(...) if it makes sense ?

Reply:
done